schemaless/create_uuid_map.py

```diff
@@ -2,11 +2,11 @@
 from collections import OrderedDict
 import csv
 from csv import DictReader
-from datetime import datetime
 import uuid
 
 from fileutils import open_file
-from create_schemaless import latest_values
+from schemaless.create_schemaless import latest_values
+from schemaless.sources import source_map
 
 
 class RecordGraph:
```

Comment:
So just to clarify, are you envisioning the RecordGraph would have nodes corresponding to records from all sources (including PTS, TCO in addition to PPTS)? And it's not necessarily true that the root record for a given project would always be a node corresponding to a PPTS PRJ (like for some project that goes from OCII --> DBI)?

Reply:
Right, Nodes are not necessarily PRJs or chlild records. This graph can/should be used to build the entire tree for UUID calculation purposes

Reply:
ok great, makes sense to me